### PR TITLE
Bumping dependencies

### PR DESCRIPTION
**Pull request by alejandrod**
Play 2.6.19
Dropwizard 4.0.3
Scala 2.12.6

### CODE REVIEW

**Comment by alejandrod** [comment]
@liorhar Do you think we can merge this? I don't see why not. Thanks!

**Comment by tzachz** [comment]
Thanks for the contribution!

I don't see a reason not to merge this, but I'm going to take a minute to go through the Dropwizard Metrics 4.0 release notes to see there are no (bad) surprises there. I'll also try testing this locally with a sample Play project. If all looks good, I'll merge.

File: build.sbt [hunk 1]
```diff
@@ -26,17 +28,18 @@ resolvers += "scalaz-bintray" at "http://dl.bintray.com/scalaz/releases"
...
+    "io.dropwizard.metrics" % "metrics-core" % dropwizardVersion,
+    "io.dropwizard.metrics" % "metrics-json" % dropwizardVersion,
+    "io.dropwizard.metrics" % "metrics-jvm" % dropwizardVersion,
+    "io.dropwizard.metrics" % "metrics-logback" % dropwizardVersion,
+    "com.typesafe.play" %% "play" % playVersion % Provided,
+    "org.joda" % "joda-convert" % "2.1.1",
+
+    //Test
+    "com.typesafe.play" %% "play-test" % playVersion % Test,
+    "com.typesafe.play" %% "play-specs2" % playVersion % Test
```
Comment (tzachz) [inline build.sbt]: thanks for the cleanup :+1:

File: src/main/scala/com/kenshoo/play/metrics/Metrics.scala [hunk 2]
```diff
@@ -67,7 +68,7 @@ class MetricsImpl @Inject() (lifecycle: ApplicationLifecycle, configuration: Con
     }
   }
 
-  def onStart() = {
+  def onStart(): ObjectMapper = {
```
Comment (tzachz) [inline src/main/scala/com/kenshoo/play/metrics/Metrics.scala]: It's not critical, but I think the return type here should be `Unit` - indeed the method's last expression has type `ObjectMapper`, but that's just coincidental, it wasn't supposed to be exposed (which just goes to show why public methods should have explicit return types, so thanks for that).

**Comment by tzachz** [comment]
New version (`2.6.19_0.7.0`) is now [published](https://oss.sonatype.org/#nexus-search;quick~metrics-play) (for both Scala 2.12 and 2.11).

Thanks again @alejandrod!

**Pull request merged**